docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 services:
   openfaas-jekyll:
     command: jekyll serve --force_polling
-    image: jekyll/jekyll:latest
+    image: jekyll/jekyll:3.8.4

```

Comment:
Latest tag doesn't build anymore:
```
docker-compose up
WARNING: The Docker Engine you're using is running in swarm mode.

Compose does not use swarm mode to deploy services to multiple nodes in a swarm. All containers will be scheduled on the current node.

To deploy your application across the swarm, use `docker stack deploy`.

Recreating openfaasgithubio_openfaas-jekyll_1 ... done
Attaching to openfaasgithubio_openfaas-jekyll_1
openfaas-jekyll_1  | /usr/gem/bin/bundle:23:in `load': cannot load such file -- /usr/local/lib/ruby/gems/2.5.0/gems/bundler-1.17.2/exe/bundle (LoadError)
openfaas-jekyll_1  |    from /usr/gem/bin/bundle:23:in `<main>'
openfaasgithubio_openfaas-jekyll_1 exited with code 1
```

So rolled back to previous version

Reply:
I'm trying the latest tag on my Intel NUC now to verify.

Reply:
looks like 3.8.5 has fixed this problem